qa/rpc-tests/wallet.py

```diff
@@ -224,5 +224,29 @@ def run_test (self):
         assert("joinSplitPubKey" not in mytxdetails)
         assert("joinSplitSig" not in mytxdetails)
 
+        # Get a new transparent address by extracting from a unified address
+        acct0 = self.nodes[0].z_getnewaccount()['account']
+        ua = self.nodes[0].z_getaddressforaccount(acct0)['address']
+        ua_taddr = self.nodes[0].z_listunifiedreceivers(ua)['p2pkh']
+
+        # Check that the ua has no funds.
+        assert_equal(
+                {'pools': {}, 'minimum_confirmations': 1}, 
+                self.nodes[0].z_getbalanceforaccount(acct0))
+        pre_balance = self.nodes[0].getbalance("*", 1, False, True)
+
+        # Send some funds to the transparent receiver of the UA
+        ua_txid = self.nodes[2].sendtoaddress(ua_taddr, 5.0)
```

Comment:
```suggestion
        self.nodes[2].sendtoaddress(ua_taddr, 5.0)
```
Fix for pyflakes lint.